content/git/concepts/add/add.md

```diff
@@ -53,5 +53,5 @@ There are many extentions to the `git add` command such as:
 
 1. `-n`: Where the file will not be added to the staging area but will show whether or not the file exists and/or if it will be ignored.
 2. `-f`: Which stages all files including those which are ignored.
-3. `.a`: Which stages all the files in the current directory.
+3. `.` : Which stages all the files in the current directory. (On the command line, the use of `.` references the directory a user is currently in.)
 4. `-A`: Which stages all the files in the current directory as well as subdirectories.
```

Comment:
I'm beginning to wonder if using an unordered list makes more sense? This is, after all, just a list of commonly used flags for `git add`, not a sequence of steps or anything like that.

Reply:
Yeah, I think that makes more sense here.